src/ck-ldb/CentralLB.C

```diff
@@ -45,20 +45,26 @@ CkGroupID loadbalancer;
 int * lb_ptr;
 bool load_balancer_created;
 
-CreateLBFunc_Def(CentralLB, "CentralLB base class")
```

Comment:
Would it make sense to add this as an (optional?) parameter to `CreateLBFunc_Def`?

Reply:
Done via #3224.